spec/inputs/eventlog_spec.rb

```diff
@@ -1,7 +1,11 @@
 require "logstash/devutils/rspec/spec_helper"
-require 'logstash/inputs/eventlog'
```

Comment:
sure you know, just write there ```if windows? then warning_to_the_user endif```. You know this will not be run without rspec being involved.